contributions.md

```diff
@@ -0,0 +1,11 @@
+# Contributions
+
+## Contributors
+We appreciate all of our contributors!
+Each contributor below has made a statement of how they feel they've contributed to `tedana`.
+- Joshua Teves helped manage issues and pull requests for a variety of both administrative and code-specific tasks.
+
+## Funding
+Special thanks to the following sources of funding for `tedana`:
+- National Institutes of Mental Health, Section on Functional Imaging Methods for supporting the 2019 `tedana` hackathon.
```

Comment:
```suggestion
- National Institutes of Mental Health, [Section on Functional Imaging Methods](https://fim.nimh.nih.gov) for supporting the 2019 `tedana` hackathon.
```

Reply:
That's true, we're doing all that work on the website, we should link to it!